Userland/Libraries/LibELF/DynamicLoader.h

```diff
@@ -40,6 +40,11 @@ enum class ShouldInitializeWeak {
     No
 };
 
+enum class ShouldCallIfuncResolver {
```

Comment:
Somewhat pre-existing, but the capitalization here bothers me. Why do we style this as `Ifunc` rather than `IFunc` ?

Reply:
Most documents I found call the feature either "IFUNC" or "ifunc" (the ABI docs, glibc wiki, etc.), so "IFunc" would look odd to my eyes.